src/skin/skinloader.cpp

```diff
@@ -72,7 +74,11 @@ QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schem
     if (preview.isNull()) {
         preview.load(":/images/skin_preview_placeholder.png");
     }
-    preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
+    QSize scaledSize(static_cast<int>(640 * devicePixelRatio),
+            static_cast<int>(360 * devicePixelRatio));
+    preview.setDevicePixelRatio(devicePixelRatio);
```

Comment:
Is this line redundant?

Reply:
nope. without line79 the image would just be as big as set by the QSize.

Reply:
whoops, you refer to the duplicate setDevicePixelRatio line above the code quote here.
yes, that's unnnecessary.
fixed.